<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>824</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>871</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请打开图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="original_btn">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>原图(O)</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>371</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="butter_high_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Butterworth高通</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="reverse_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>彩色负片</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="vena_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>维纳滤波</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="mean_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>均值滤波</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="gauss_low_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>高斯低通</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="inverse_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>逆滤波</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="mid_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>中值滤波</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="ostu_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Otsu</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="gauss_high_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>高斯高通</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="fft_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>傅立叶变换频谱</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="gamma_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>伽马校正</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="hist_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>直方图均衡化</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="oil_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>油画</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="lap_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>拉普拉斯</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="enhance_sharp_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>增强锐度</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="sketch_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>素描</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="butter_low_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Butterworth低通</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="exp_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>指数灰度</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="enhance_color_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>增强颜色</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="enhance_bright_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>增强亮度</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="enhance_contrast_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>增强对比度</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="edge_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>边缘提取</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>710</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="previous">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>上一张(A)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>打开图片(P)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="next">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>下一张(D)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
